tests/cxx/builtins_new_included.cc

```diff
@@ -1,4 +1,4 @@
-//===--- built_ins_new_included.cc - test input file for iwyu -------------===//
+//===--- builtins_new_included.cc - test input file for iwyu --------------===//
```

Comment:
Sorry I tricked you into this :)

Can you revert for now, and we can clean it up separately? Thanks